testsuite/MDAnalysisTests/transformations/test_translate.py

```diff
@@ -0,0 +1,169 @@
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+from __future__ import absolute_import
+
+import numpy as np
+import pytest
+from numpy.testing import assert_array_almost_equal
+
+import MDAnalysis as mda
+from MDAnalysis.transformations import translate, center_in_box
+from MDAnalysisTests import make_Universe
+
+@pytest.fixture()
+def translate_universes():
+    # create the Universe objects for the tests
+    reference = make_Universe(trajectory=True)
+    transformed = make_Universe(['masses'], trajectory=True)
+    transformed.trajectory.ts.dimensions = np.array([372., 373., 374., 90, 90, 90])
+    return reference, transformed
+
+def test_translate_coords(translate_universes):
+    ref_u = translate_universes[0]
+    trans_u = translate_universes[1]
+    ref = ref_u.trajectory.ts
+    vector = np.float32([1, 2, 3])
+    ref.positions += vector
+    trans = translate(vector)(trans_u.trajectory.ts)
+    assert_array_almost_equal(trans.positions, ref.positions, decimal=6)
+
+def test_translate_vector(translate_universes):
+    # what happens if the vector argument is smaller than 3?
+    ts = translate_universes[0].trajectory.ts
+    vector = [0, 1]
+    with pytest.raises(ValueError):
+        translate(vector)(ts)
+    # what happens if the vector argument is bigger than 3?
+    vector = [0, 1, 2, 3]
+    with pytest.raises(ValueError):
+        translate(vector)(ts)
+
+def test_center_in_box_bad_ag(translate_universes):
+    # this universe as a box size zero
+    ts = translate_universes[0].trajectory.ts
+    ag = translate_universes[0].residues[0].atoms
+    # what happens if something other than an AtomGroup is given?
+    bad_ag = 1
+    with pytest.raises(ValueError): 
+        center_in_box(bad_ag)(ts)
+
+def test_center_in_box_bad_point(translate_universes):
+    # this universe as a box size zero
+    ts = translate_universes[0].trajectory.ts
+    ag = translate_universes[0].residues[0].atoms
+    # what if the box is in the wrong format?
+    bad_point = [1]
+    with pytest.raises(ValueError): 
+        center_in_box(ag, point=bad_point)(ts)
+    
+def test_center_in_box_bad_pbc(translate_universes):    
+    # this universe as a box size zero
+    ts = translate_universes[0].trajectory.ts
+    ag = translate_universes[0].residues[0].atoms
+    # is pbc passed to the center methods?
+    # if yes it should raise an exception for boxes that are zero in size
+    with pytest.raises(ValueError): 
+        center_in_box(ag, pbc=True)(ts)
+
+def test_center_in_box_bad_center(translate_universes):
+    # this universe as a box size zero
+    ts = translate_universes[0].trajectory.ts
+    ag = translate_universes[0].residues[0].atoms
+    # what if a wrong center type name is passed?
+    bad_center = " "
+    with pytest.raises(ValueError): 
+        center_in_box(ag, center=bad_center)(ts)
+    
+def test_center_in_box_no_masses(translate_universes):   
+    # this universe as a box size zero
```

Comment:
This is not the comment you care about. What matters here is that the universe does not have masses.